package-lock.json

```diff
@@ -2815,28 +2815,28 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```

Comment:
Not sure why `package-lock.json` needs to change 🤔 
cc @sblatz any thoughts on this?

Reply:
Agreed, think this is a random change that got pulled in.